frontend/README.md

```diff
@@ -66,6 +66,51 @@ Each component file needs to host a single component and needs to have the follo
 - component props interface with the name `${ComponentName}Props` (if needed)
 - functional component declaration
 
+Additionally:
+
+- if a task can be **_easily_** implemented using tailwind/css follow that implementation instead of relying on React/TS
```

Comment:
```suggestion
- if a task (like manipulating dynamic elements, animations, resizing) can be **_easily_** implemented using tailwind/css follow that implementation instead of relying on React/TS
```